src/System.IO.Pipelines/src/System/IO/Pipelines/PipeCompletion.cs

```diff
@@ -17,15 +17,15 @@ internal struct PipeCompletion
         private const int InitialCallbacksSize = 1;
 
         private bool _isCompleted;
-        private ExceptionDispatchInfo _exceptionInfo;
+        private Exception _exception;
 
         private PipeCompletionCallback _firstCallback;
         private PipeCompletionCallback[] _callbacks;
         private int _callbackCount;
 
         public bool IsCompleted => _isCompleted;
 
-        public bool IsFaulted => _exceptionInfo != null;
+        public bool IsFaulted => _exception is object;
```

Comment:
Why is this better?  We've generally opted not to use this style across corefx; are we using it in pipelines everywhere?

Reply:
`is object` is the future 😉 /cc @agocke 

Will revert back

Reply:
> `is object` is the future

I hope not.  I personally find the syntax confusing for what it does.
https://github.com/dotnet/csharplang/issues/1944

Reply:
From the issue, looks like there may be an even better future in the works then ;)

Reply:
Just leaving this here 😉 

![image](https://user-images.githubusercontent.com/1142958/60756273-775d8e00-9ff3-11e9-9d99-47b951bfb800.png)

Reply:
> Just leaving this here

You'll be more likely to get traction on bugs like this if you open a separate issue :wink: